gui/velociraptor/src/components/vfs/browse-vfs.jsx

```diff
@@ -78,7 +78,7 @@ class VFSViewer extends Component {
             </SplitPane>
             </>
         );
-    };
```

Comment:
Heh I can never remember when ; are needed and when they are not so I just add them everywhere randomly until the editor shows green :-)

Doesn't seem like it is something we need to worry about too much?

I actually don't really use the lint, instead just relying on the editor but i guess it is useful.

Reply:
That's fair, my thought process was trying to match the go tooling on the js side with this PR.

Reply:
No this is great! Thank you - we should make it easy to do the right thing - looks like we can just do make lint here and it will point out the issues.